DonjinKrawler-Server/src/main/java/donjinkrawler/enemy_generation/EnemyFactory.java

```diff
@@ -0,0 +1,21 @@
+package donjinkrawler.enemy_generation;
+
+public class EnemyFactory {
+
+    public Enemy make(String type) {
+
+        if(type == null) {
+            return null;
+        }
+
+        if(type.equalsIgnoreCase("CHICKEN")) {
```

Comment:
This would be a great use-case for the new switch rules introduced in JDK 14, if only someone didn't have a fricked up setup...

Reply:
Also, instead of using magic strings, maybe create an enum EnemyType? Could be reused in EnemyGenerator as well.